docs/core/compatibility/7.0.md

```diff
@@ -24,3 +24,9 @@ If you're migrating an app to .NET 7, the breaking changes listed here might aff
 | Title | Binary compatible | Source compatible | Introduced |
 | - | - | - | - |
 | [Deserialize Version type with leading or trailing whitespace](serialization/7.0/deserialize-version-with-whitespace.md) | ❌ | ✔️ | Preview 1 |
+
+## Windows Forms
+
+| Title | Binary compatible | Source compatible | Introduced |
+| - | - | - | - |
+| [Some APIs throw ArgumentNullException](windows-forms/7.0/apis-throw-argumentnullexception.md) | ❌ | ✔️ | Preview 1 |
```

Comment:
Not really related to this PR, but this gives me an OCD...
![image](https://user-images.githubusercontent.com/4403806/149681699-d654570d-d9d6-4619-801b-3a212873e649.png)

If possible I recommend [center aligning](https://markdown.land/markdown-table#aligning-columns) the images (2nd and 3rd columns).
It'd also be great to "fix" the width of the last three columns too. Unfortunately MD syntax does not provide a way to do this, but looks like there are few possible clever ways of achieving this (e.g. [this](https://stackoverflow.com/questions/36121672/set-table-column-width-via-markdown)).

Reply:
Good suggestion on the centering. I tried the div tag, but got a linter error so took it out. I think the many &nbsp; is just too ugly in the raw Markdown.

Reply:
> Good suggestion on the centering. I tried the div tag, but got a linter error so took it out. 

Do you know who owns the linter? May be the rules can be tweaked or exclusions added?

> I think the many   is just too ugly in the raw Markdown.

I think we should prioritise the end user experience, but I agree that adding "&nbsp;" may be a non scalable maintenance burden.